src/shared/install.c

```diff
@@ -56,6 +56,9 @@
 
 #define UNIT_FILE_FOLLOW_SYMLINK_MAX 64
 
+#define PRESET_ENABLE  1
+#define PRESET_DISABLE 2
+
```

Comment:
Actually why not make this a non-anonymous enum? Then it can be used below 'int action' → 'EnableAction action'.